static/js/components/RecentSources.jsx

```diff
@@ -24,16 +24,18 @@ export const useSourceListStyles = makeStyles((theme) => ({
   stampContainer: {
     display: "contents",
   },
-  stamp: {
+  stamp: (props) => ({
```

Comment:
I'm surprised the linter did not make you unpack that into `({invertThumbnails})`

Reply:
@stefanv Good point. There were some other style rules already doing the same thing in ThumbnailList.jsx so I was just mindlessly following haha. Now fixed